FoodBookApp/UI/Views/Content/ContentView.swift

```diff
@@ -145,11 +150,26 @@ struct ContentView: View {
                     print("Network is offline, using fallback")
                     self.isFetching = true
                     self.model.fallback()
-                    self.isFetching = false
+                    let currentTime = Date()
+                    if let lastAlertTime = lastAlertTime, currentTime.timeIntervalSince(lastAlertTime) < 10 {
```

Comment:
Question: didn't @mar-cas3 say to change it to 30? 🤔

Reply:
right sorryyyyy ill change it rn